### DISCUSSION TITLE
Hi, how do I know the file has been successfully installed and is working?

### DISCUSSION THREAD

**Discussion by sonchemjo999** [mintyYuki/antiddos#81] [Q&A] [answered]
Please help me.

**Comment by mintyYuki** [comment, marked as answer]
Hello, try this command -> "sudo nft list ruleset". You should see many rules in the output.
Also, you can check the sysctl.conf file (path: /etc/sysctl.conf), to see whether it's modified or not.

If you want to verify if is it working - you can use the first command. It shows statistics for rules as well, you'll see how many packets and bytes were dropped by each rule.
